pkg/kv/batch.go

```diff
@@ -53,6 +53,9 @@ type Batch struct {
 	// BatchRequest. To be modified directly.
 	AdmissionHeader kvpb.AdmissionHeader
 	reqs            []kvpb.RequestUnion
+	// IsBulkFeed specifies whether this batch is a bulk scan request initiated by
+	// a changefeed initial scan.
+	IsBulkFeed bool
```

Comment:
We could replace this with a check that `len(reqs) == 1` and `reqs[0]` is a `ScanRequest`.

Then we would add a bool or some more generic hint to the `ScanRequest` proto.

Reply:
As discussed on Slack, we'll probably want to add a `ConnectionClass` field in the batch request header, in case the request needs to propagate across the network (consider e.g. follower routing which will proxy via a different node's DistSender).

Reply:
Done, PTAL.